src/blocks/homepage-articles/templates/article.php

```diff
@@ -71,11 +71,18 @@ class="post-has-image"
 				</div>
 				<?php
 			endif;
-			if ( '' === $attributes['sectionHeader'] ) :
+
+			$is_section_header = '' === $attributes['sectionHeader'];
+			if ( $is_section_header ) :
 				the_title( '<h2 class="entry-title"><a href="' . esc_url( get_permalink() ) . '" rel="bookmark">', '</a></h2>' );
 			else :
 				the_title( '<h3 class="entry-title"><a href="' . esc_url( get_permalink() ) . '" rel="bookmark">', '</a></h3>' );
 			endif;
+			?>
+			<div class="newspack-post-subtitle newspack-post-subtitle--in-homepage-block <?php echo $is_section_header ? 'newspack-post-subtitle--small' : '' ?>">
+				<?php echo get_post_meta(get_the_ID(), 'newspack_post_subtitle', true); ?>
```

Comment:
```suggestion
				<?php echo esc_html( get_post_meta( get_the_ID(), 'newspack_post_subtitle', true ) ); ?>
```

[Same as the other PR](https://github.com/Automattic/newspack-theme/pull/635/files#r359095544), this output needs some escaping. 

This bit here also needs some space to follow PHPCS rules. It was mentioned at the other PR also: you can just do `composer install` to get PHPCS automatically installed and set up, and it'll run and analyze the code you're about to commit whenever you try and commit.